packages/descendants/src/Descendants.stories.tsx

```diff
@@ -194,3 +203,63 @@ export const MultipleContexts = () => {
     </TestParent>
   );
 };
+
+const Parent = ({
+  children,
+  open,
+  ...rest
+}: ComponentProps<'div'> & { open: boolean }) => {
+  const elRef = useRef(null);
+  const parentRef = useRef(null);
+  const { descendants, dispatch, getDescendants } =
+    useInitDescendants<HTMLDivElement>();
+
+  const handleTransition = () => {
+    console.log(getDescendants().map(d => d.id));
+  };
+
+  return (
+    <>
+      <span ref={elRef} />
+      <DescendantsProvider
+        context={TestDescendantContext}
+        descendants={descendants}
+        dispatch={dispatch}
+      >
+        <Popover
+          refEl={elRef}
+          style={{ border: '1px solid red' }}
+          active={open}
+          onEntered={handleTransition}
+        >
+          <div {...rest} ref={parentRef} data-testid="parent">
+            {children}
+          </div>
+        </Popover>
+      </DescendantsProvider>
+    </>
+  );
+};
+
+export const WithPopover = {
+  render: ({ open }) => {
+    return (
+      <Parent open={open}>
+        <TestDescendant data-testid="child" tabIndex={0}>
+          Apple
+        </TestDescendant>
+        <TestDescendant data-testid="child" tabIndex={0}>
```

Comment:
do we need `data-testid`? if so, maybe use unique test ids?

Reply:
nah, I just copied it 👍